ui/src/shared/components/TimeMachine/TimeMachine.tsx

```diff
@@ -318,9 +322,11 @@ class TimeMachine extends PureComponent<Props, State> {
     return (
       <FluxQueryMaker
         notify={notify}
+        draftScript={draftScript}
```

Comment:
Instead of connecting the `draftScript` and `onChangeDraftScript` to the `TimeMachine` and passing it down to the `FluxQueryMaker`, we could instead connect them to the `FluxQueryMaker` directly (that's the advantage of the `TimeMachineContainer`). But that's minor, I'm totally fine with leaving this as is.